add-machine-ips.sh

```diff
@@ -14,14 +14,12 @@ oc --config ocp/auth/kubeconfig proxy &
 proxy_pid=$!
 
 for node in $(oc --config ocp/auth/kubeconfig get nodes -o template --template='{{range .items}}{{.metadata.uid}}:{{.metadata.name}}{{"\n"}}{{end}}'); do
-    node_name=$(echo $node | cut -f2 -d':')
     machine_name=$CLUSTER_NAME-$node_name
+    node_name=$(echo $node | cut -f2 -d':')
```

Comment:
did you mean to move this after machine_name?  machine_name needs $node_name already set

Reply:
What is "testing"? :-)

Fixed.